Source/Core/Common/IniFile.cpp

```diff
@@ -112,7 +113,7 @@ bool IniFile::Section::GetLines(std::vector<std::string>* lines, const bool remo
       }
     }
 
-    lines->push_back(std::move(stripped_line));
+    lines->push_back(std::string(stripped_line));
```

Comment:
I must have previously overlooked that, but it can be just
`lines->emplace_back(stripped_line);`
Both should be nearly identical though so maybe don't bother.